pages/profile/read/content/index.js

```diff
@@ -1,27 +1,16 @@
 module.exports = {
   profile: 'Profile',
   responsibilities: {
-    title: 'Responsibilities',
-    roles: 'Roles',
-    noRoles: 'This person has no named roles',
-    roleApply: 'Apply for named role',
-    roleRemove: 'Remove a named role'
-  },
-  qualifications: 'Qualifications',
-  training: {
-    title: 'Training',
-    nacwo: 'NACWO training',
-    modulesTitle: 'Accredited training modules',
-    module: 'Module {{module}}',
-    dateCompleted: 'Date completed: {{date}}'
+    title: 'Roles',
+    noRoles: "You don't have any named roles",
```

Comment:
`'` not `"`, not sure why lint isn't picking this up 🤔

Reply:
done

Reply:
much better - just a couple of random styles then we're g2g